src/account-manager.js

```diff
@@ -199,23 +199,30 @@ class AccountManager {
         // TODO: Remove unreturned addresses and add new returned addresses
     }
 
-    async export(accountId) {
+    async exportFile(accountId) {
         await this._launched;
-        const result = await this._invoke('export', null, {
+        const result = await this._invoke('exportFile', null, {
             appName: 'Nimiq Safe',
             accountId,
         });
 
-        // Update hasFile/hasWords flags
-        const wallet = MixinRedux.store.getState().wallets.entries.get(accountId);
-        if (!wallet) return;
-        const updatedWallet = Object.assign({}, wallet, {
-            hasFile: result.fileExported,
-            hasWords: result.wordsExported,
+        if (result.success) {
```

Comment:
Additionally, the `success` property is not returned by the export result. The result contains flags which types have been exported, just like in the code that you removed here.

https://github.com/nimiq/accounts/commit/4d29435e5e4490401a3c86a90788db0114b4ebfa#diff-20abd8c6e7eeac80d4ea800026b6cccaR92